lib/simplifyExpression/basicsSearch/convertMixedNumberToImproperFraction.js

```diff
@@ -0,0 +1,138 @@
+const clone = require('../../util/clone');
+const mixedNumber = require('./mixedNumber');
+
+const ChangeTypes = require('../../ChangeTypes');
+const Node = require('../../node');
+
+// Converts a mixed number to an improper fraction
+// e.g. 1 2/3 -> 5/3
+function convertMixedNumberToImproperFraction(node) {
+  if (!mixedNumber.isMixedNumber(node)) {
+    return Node.Status.noChange(node);
+  }
+
+  const substeps = [];
+  let newNode = clone(node);
+
+  // e.g. 1 2/3
+  const wholeNumber = mixedNumber.getWholeNumber(node); // 1
+  const numerator = mixedNumber.getNumerator(node); // 2
+  const denominator = mixedNumber.getDenominator(node); // 3
+  const isNegativeMixedNumber = mixedNumber.isNegativeMixedNumber(node);
+
+  // STEP 1: Convert to unsimplified improper fraction
+  // e.g. 1 2/3 -> ((1 * 3) + 2) / 3
+  let status = convertToUnsimplifiedImproperFraction(
+    newNode, wholeNumber, numerator, denominator, isNegativeMixedNumber);
+  substeps.push(status);
+  newNode = clone(status.newNode);
```

Comment:
Ah, I had to do this cloning because I wasn't setting the change group on the old node, as per your comment below! But now I'm setting it on the old nodes, so I do need to reset them there. (Though my previous approach also works -- the change group *does* get set on the old node because the old node is really just the previous new node, which did have a change group, and if you don't reset them, it just gets passed down ... if that makes sense)